test/integration/targets/nmcli/aliases

```diff
@@ -0,0 +1,5 @@
+destructive
+posix/ci/group1
+skip/freebsd
+skip/osx
+systemd
```

Comment:
Why was the `systemd` alias added here? The `systemd` module already a test target.

Reply:
I thought if I add the systemd to alias the test will run without docker.